examples/jsm/renderers/webgpu/WebGPUBackground.js

```diff
@@ -154,6 +156,7 @@ class WebGPUBackground {
 			} else {
 
 				depthStencilAttachment.stencilLoadOp = GPULoadOp.Load;
+				depthStencilAttachment.stencilLoadOp = GPUStoreOp.Store;
```

Comment:
Assigning two values to `depthStencilAttachment.stencilLoadOp` does not look right.

Reply:
Thanks for noticing that :)